onnxruntime/contrib_ops/rocm/diffusion/group_norm_common.h

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <string>
+
+#include "core/providers/rocm/rocm_common.h"
+#include "core/providers/rocm/tunable/rocm_tunable.h"
+
+namespace onnxruntime {
+namespace contrib {
+namespace rocm {
+
+static inline int32_t divUp(int32_t m, int32_t n) {
+  return (m + n - 1) / n;
+}
+
+int32_t findMaxDivisor(int32_t n, int32_t maxAllowedDivisor) {
```

Comment:
Also, the naming style should be changed to conform with the guideline .

Reply:
If most of them are copied, then lets leave it unchanged for now

Reply:
Yes, Most of them are copied.